core/runtime/bindings.rs

```diff
@@ -243,47 +242,42 @@ pub(crate) fn initialize_context<'s>(
 
   let global = context.global(scope);
 
-  let mut codegen = String::with_capacity(op_ctxs.len() * 200);
-  codegen.push_str(include_str!("bindings.js"));
-  _ = writeln!(codegen, "Deno.__op__ = function(opFns) {{");
+  let deno_obj = get(scope, global, b"Deno", "Deno");
+  let deno_core_obj = get(scope, deno_obj, b"core", "Deno.core");
+  let deno_core_ops_obj: v8::Local<v8::Object> =
```

Comment:
Add comment what's going on in this function.

Also this function should be combined with `initialize_deno_namespace_object` and execute `00_primordials.js` and `00_infra.js`.